tool/tctl/common/resource_command.go

```diff
@@ -654,9 +654,18 @@ func (rc *ResourceCommand) createNode(ctx context.Context, client auth.ClientI,
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	if err := server.CheckAndSetDefaults(); err != nil {
+		return trace.Wrap(err)
+	}
 
-	if !rc.force {
-		return trace.AlreadyExists("nodes cannot be created, only upserted")
+	name := server.GetName()
+	_, err = client.GetNode(ctx, server.GetNamespace(), name)
```

Comment:
How do we handle name conflicts btw? Since name is UUID - can you register the same OpenSSH mode multiple times under different names?

Reply:
If you mean register multiple OpenSSH nodes with the same Addrs/Hostnames but different UUIDs, currently we can